ansible/roles/test/tasks/buffer_profile.yml

```diff
@@ -0,0 +1,36 @@
+#
+# ansible-playbook test_sonic.yml -i inventory --limit arc-switch1028 --tags buffer_profile -e skip_interface_check=true
```

Comment:
what is the plan here? this test is cfggen test, seems like a unit test?

Reply:
yes, currently it tests buffers profile selection for all possible combinations of port speed+cable length
there were also tests for run-time profile change - I removed them until we agree and implement the functional part